src/pages/signup/Signup.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { Col, Container, Row, Spinner } from 'reactstrap';
+import { Redirect, RouteProps } from 'react-router-dom';
+
+import AuthenticationHeader from 'components-work/AuthenticationHeader';
+import AuthenticationTestimonials from 'components-work/AuthenticationTestimonials';
+import { FirebaseConsumer, FirebaseUserProps } from 'HOCs/FirebaseProvider';
+
+import SignupForm from './SignupForm';
+
+const Signup = (props: RouteProps) => (
+  <FirebaseConsumer>
+    {({ loading, user }: FirebaseUserProps) => {
+      if (loading) {
+        return (
+          <Container className="d-flex align-items-center justify-content-center position-relative height-100vh">
```

Comment:
need to use styled components to make reusable containers.  will do this in separate pr

Reply:
addressed as string variable